std/datetime.d

```diff
@@ -16153,6 +16153,21 @@ public:
         Params:
             duration = The duration to add to or subtract from this
                        $(LREF DateTime).
+
+        See_Also:
+            $(CXREF time, Duration).
```

Comment:
Yes, that is better. I'm not a Git expert. Is there a way I can edit this pull request, or do I have to close it and create a new one?